app/src/main/java/net/skyscanner/backpack/demo/ThemeApplier.kt

```diff
@@ -25,14 +25,14 @@ import net.skyscanner.backpack.demo.data.SharedPreferences
 import net.skyscanner.backpack.util.BpkTheme
 
 object ThemeApplier : Application.ActivityLifecycleCallbacks {
-  override fun onActivityPaused(activity: Activity?) {}
-  override fun onActivityResumed(activity: Activity?) {}
-  override fun onActivityStarted(activity: Activity?) {}
-  override fun onActivityDestroyed(activity: Activity?) {}
-  override fun onActivitySaveInstanceState(activity: Activity?, outState: Bundle?) {}
-  override fun onActivityStopped(activity: Activity?) {}
-  override fun onActivityCreated(activity: Activity?, savedInstanceState: Bundle?) {
-    activity?.theme?.applyStyle(SharedPreferences.getTheme(activity), true)
-    activity?.let { BpkTheme.applyDefaultsToContext(it) }
+  override fun onActivityPaused(activity: Activity) {}
```

Comment:
Nullability info was added to ActivityLifecycleCallbacks. It states Activity is never null in these methods.